sirepo/job_driver/sbatch.py

```diff
@@ -211,12 +211,13 @@ def _raise_sbatch_login_srexception(self, reason, msg):
         raise util.SRException(
             'sbatchLogin',
             PKDict(
+                host=self.cfg.host,
                 isModal=True,
+                nonJobExc=True,
```

Comment:
Tough name. `isGeneral` matches style of `isModal`. The word "general" can be overused, but "nonJob" seems strange to me.

Reply:
Tough name indeed. isGeneral seems to general (har har har) but it matches isModal and nonJob is no better. Changed https://github.com/radiasoft/sirepo/pull/2523/commits/2c505a1eca579dafbef6921f946a8bc6a642cdc8